sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -674,3 +676,61 @@ def validate_cache_staleness_value(max_integrated_cache_staleness):
     if max_integrated_cache_staleness < 0:
         raise ValueError("Parameter 'max_integrated_cache_staleness_in_ms' can only be an "
                          "integer greater than or equal to zero")
+
+
+def _stringify_auto_scale(offer: Dict[str, Any]) -> Any:
+    auto_scale_params = None
+    max_throughput = offer.auto_scale_max_throughput
+    increment_percent = offer.auto_scale_increment_percent
+    if max_throughput is not None:
```

Comment:
We should remove this line - as this check is already on line 700.
It also means if a user has specified `increment_percent` but not `max_throughput` they will get a `NameError` on line 688.